### ISSUE TITLE
FullDebugMode under Lazarus

### ISSUE THREAD

**Issue by TetrisSQC** [maximmasiutin/FastMM4-AVX#33]
Do you see a tweak to allow FullDebugMode under Lazarus? 

Christian

**Comment by maximmasiutin** [comment]
Yes, in the main workflow here on GigHub there are many cases of using FullDebugMode under FreePascal.
